src/NIST-800-53/rules/DynamoDB/nist80053DynamoDBTableEncryptedKMS.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
I don't think we need this rule.

From the [CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-table-ssespecification.html)

SSEEnabled:
> Indicates whether server-side encryption is done using an AWS managed CMK or an AWS owned CMK. If enabled (true), server-side encryption type is set to KMS and an AWS managed CMK is used (AWS KMS charges apply). If disabled (false) or not specified, server-side encryption is set to AWS owned CMK. 

A DynamoDB table is always encrypted with some sort of KMS encryption, the check is not distinguishing between  an AWS managed CMK or an AWS owned CMK.